grpc_test.go

```diff
@@ -620,3 +620,19 @@ func TestMultiWaitProcess(t *testing.T) {
 
 	wg.Wait()
 }
+
+func TestQueryResource(t *testing.T) {
+	assert := assert.New(t)
+	a := &agentGRPC{
+		sandbox: &sandbox{
+			containers: make(map[string]*container),
+		},
+	}
+
+	req := &pb.QueryResourceRequest{
+		MemBlockSize: true,
+	}
+
+	_, err := a.QueryResource(context.TODO(), req)
```

Comment:
How about also comparing the return value with `/sys/devices/system/memory/block_size_bytes`?

Reply:
I think about this too but it's dispensable. I will update it.